TenantFile/Api/tenant-file-12264437b25e.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```

Comment:
This needs to be deleted, I disabled the service account associated with it.

Reply:
Thanks for catching this! I had meant to exclude this from the PR as well.

Reply:
I'd recommend storing the file outside this project. I store mine in ~/.gcloud/keys.json

Reply:
Thanks, will do!